dbt/include/firebolt/macros/adapters.sql

```diff
@@ -184,32 +195,42 @@
 {% endmacro %}
 
 
-{% macro firebolt__create_table_as(temporary, relation, sql) -%}
+{% macro firebolt__create_table_as(temporary, relation, ctas_sql) -%}
   {# Create table using CTAS
      Args:
       temporary (bool): Unused, included so macro signature matches
           that of dbt's default macro
       relation (dbt relation/dict):
```

Comment:
Missing relation description and `ctas_sql` arg

Reply:
Ooh, good catch! The formatting was actually a little off, too.